packages/package-json-lint-config-terra/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@cerner/package-json-lint-config-terra",
+  "version": "0.0.0",
+  "private": true,
```

Comment:
Not sure if you wanna remove this now or later

Reply:
we remove it in the first release PR